backend/api/src/markets.ts

```diff
@@ -35,37 +35,47 @@ const getPublicContracts = async (
     beforeTime?: string
     order?: 'asc' | 'desc'
     userId?: string
+    groupId?: string
   }
 ) => {
-  let q = selectJson(db, 'public_contracts')
-  q = q.order('created_time', {
+  const q = selectJson(db, 'public_contracts')
+  q.order('created_time', {
```

Comment:
I thought these queries were immutable and returned new instances? But here it looks like it is mutating the query.